enhancements/sig-architecture/58-addon-configuration/README.md

```diff
@@ -0,0 +1,439 @@
+	// +optional
+	Tolerations []v1.Toleration `json:"tolerations,omitempty"`
+}
+```
+
+Example:
+
+We have a default add-on deployment configuration on the hub
+
+```yaml
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: AddOnDeployementConfig
+metadata:
+  name: addon-default-placement
+spec:
+  nodeSelector:
+    "kubernetes.io/arch": "amd64"
+  tolerations:
+  - key: "node.kubernetes.io/unreachable",
+    operator: "Exists",
+    effect: "NoSchedule"
+```
+
+User can configure the `ConfigCoordinates` in the `ClusterManagementAddOn` to share this configuration to their add-ons
+
+```yaml
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ClusterManagementAddOn
+metadata:
+  name: myaddon
+spec:
+  addOnConfiguration:
+    group: addon.open-cluster-management.io
+	version: v1alpha1
+    resource: ManagedClusterAddOn
+    defaultName: addon-default-placement
+```
+
+For each managed cluster add-on, the configuration can be located with `ConfigReference` in the `ManagedClusterAddOnStatus`
+
+```yaml
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ManagedClusterAddOn
+metadata:
+  name: myaddon
+  namespace: cluster1
+spec:
+  installNamespace: open-cluster-management-agent-addon
+status:
+  configReference:
+    group: addon.open-cluster-management.io
+	version: v1alpha1
+    resource: ManagedClusterAddOn
+	name: addon-default-placement
+	lastObservedGeneration: 1
+
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ManagedClusterAddOn
+metadata:
+  name: myaddon
+  namespace: cluster2
+spec:
+  installNamespace: open-cluster-management-agent-addon
+status:
+  configReference:
+    group: addon.open-cluster-management.io
+	version: v1alpha1
+    resource: ManagedClusterAddOn
+	name: addon-default-placement
+	lastObservedGeneration: 1
+```
+
+
+If user need to configure some add-ons specially, the use can define a specifc configuration and specify the
+configuration name in the `ManagedClusterAddOn`
+
+```yaml
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: AddOnDeployementConfig
+metadata:
+  name: addon-arm-placement
+spec:
+  nodeSelector:
+    "kubernetes.io/arch": "arm64"
+  tolerations:
+  - key: "node.kubernetes.io/unreachable",
+    operator: "Exists",
+    effect: "NoSchedule"
+
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ManagedClusterAddOn
+metadata:
+  name: myaddon
+  namespace: cluster3
+spec:
+  installNamespace: open-cluster-management-agent-addon
+  configuration:
+    name: addon-arm-placement
+```
+
+then the configuration reference in this add-on status will be
+
+```yaml
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ManagedClusterAddOn
+metadata:
+  name: myaddon
+  namespace: cluster3
+spec:
+  installNamespace: open-cluster-management-agent-addon
+  configuration:
+    name: addon-arm-placement
+status:
+  configReference:
+    group: addon.open-cluster-management.io
+	version: v1alpha1
+    resource: ManagedClusterAddOn
```

Comment:
the resource should be AddOnDeployementConfig

Reply:
corrected